src/Services/Payment/Payment.API/Startup.cs

```diff
@@ -130,9 +128,11 @@ private void RegisterEventBus(IServiceCollection services)
                     var iLifetimeScope = sp.GetRequiredService<ILifetimeScope>();
                     var logger = sp.GetRequiredService<ILogger<EventBusServiceBus>>();
                     var eventBusSubcriptionsManager = sp.GetRequiredService<IEventBusSubscriptionsManager>();
+                    string topicName = "topicName";
```

Comment:
I don't think we can just hardcode these values. 
SubscriptionName would come from the subscriptionClientName config.
topicName should be in the connection string.

To extract the topicName, we can use ServiceBusConnectionStringProperties.Parse